src/main/java/dev/aziz/grocerystore/entities/User.java

```diff
@@ -0,0 +1,42 @@
+package dev.aziz.grocerystore.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Data
+@Entity
+@Table(name = "app_user")
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@EntityListeners(AuditingEntityListener.class)
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull(message = "Firstname should not be empty")
+    private String firstName;
+
+    @NotNull(message = "Lastname should not be empty")
+    private String lastName;
+
+    @NotNull(message = "Login should not be empty")
+    private String login;
+
+    @NotEmpty(message = "Password should not be empty")
+    private char[] password;
```

Comment:
You can store the password as a String, because it's not the real password which will be stored but a crypted version.